docs/pages/management/operations/proxy-peering.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Proxy Peering Migration
+description: How to upgrade an existing Teleport cluster to proxy peering mode.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.0"
+>
+  TLS routing is available starting from Teleport `10.0`.
+</Details>
+
+## Requirements
```

Comment:
Are there any other requirements? E.g., is this guide for an organization with a pre-existing self-hosted Teleport cluster?

Should the cluster already be configured to use multiple Proxy Service instances?

Reply:
I will update the requirements for having an existing teleport cluster.

As for multiple proxy service instances it is not a requirement, however there wouldn't be a reason to enable proxy peering with a single proxy instance. So what do you think about some guidance on the benifits/use cases for proxy peering above this section?